worldmap.js

```diff
@@ -0,0 +1,72 @@
+const countries = document.getElementById("countries");
+countries.setAttribute("id", "countries")
+
+
+//Returns an array with all countries, sorted by population
+async function sortByPopulation() {
+    const API = await axios.get("https://restcountries.eu/rest/v2/all");
+    const arrayPopulations=[];
+    for (let country of API.data) {
+        const {name, flag, population, region} = country;
+        arrayPopulations.push({name,flag,population,region});
+    }
+    arrayPopulations.sort((a,b) => {
+        return a.population-b.population;
+    });
+    return arrayPopulations;
+}
+
+//Shows all countries on the page in corresponding colors
+async function showCountries() {
+    const countriesList = await sortByPopulation();
+        for (let country of countriesList) {
+            const flag = document.createElement("img");
+            flag.setAttribute("src", country.flag);
+            flag.setAttribute("id", "flagmap");
+            const newItem = document.createElement("div");
+            const newItemPopulation = document.createElement("div");
+            newItemPopulation.setAttribute("id", "population-"+(country.name));
+            newItem.append(flag, country.name);
```

Comment:
`cursor: pointer` is een leuke toevoeging zodat je visueel ziet dat je ergens op kan klikken